transaction/models.py

```diff
@@ -17,6 +18,15 @@ class TransactionStaticData(models.Model):
                             help_text='The type of transaction (income or expense)')
     description = models.TextField(help_text='A short description of what the transaction is for')
     notes = models.TextField(blank=True, help_text='A longer description and details about the transaction')
+    categories = models.ForeignKey(
+        Category,
+        unique=False,
+        blank=True,
+        null=True,
+        help_text='Transaction categories',
+        on_delete=models.DO_NOTHING,
```

Comment:
Why `DO_NOTHING`? This will leave the transaction pointing to a non-existing category if it's deleted, causing problems for both frontend and backend

Reply:
I had not noticed `SET_NULL` which seems more appropriate; the idea was to not delete transactions on removing a category, which seemed to be the default behavior.